app/frontend/src/components/forms/TeamManagement.vue

```diff
@@ -97,8 +104,23 @@ export default {
     addNewUsers(users) {
       if (Array.isArray(users) && users.length) {
         users.forEach((user) => {
-          // TODO: Create a new this.tableData object and add it in
-          console.log(user); // eslint-disable-line no-console
+          // if user isnt already in the table
+          if(this.tableData.filter(function(obj) { return obj.userId === user.id; }).length < 1) {
```

Comment:
`Array.prototype.some()` is a better function to use in this case
```js
if(!this.tableData.some((obj) => obj.userId === user.id)) {
```
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some